interfaces/html.idl

```diff
@@ -2048,6 +2018,10 @@ interface mixin NavigatorLanguage {
   readonly attribute FrozenArray<DOMString> languages;
 };
 
+interface mixin NavigatorOnLine {
+  readonly attribute boolean onLine;
+};
```

Comment:
This seems duplicated now?

Reply:
Thanks for the catch, I had messed up copying from https://github.com/web-platform-tests/wpt/pull/26692/files

(It was meant to be moved, but I just duplicated it instead -_-)